.github/ISSUE_TEMPLATE/feature_request_template.md

```diff
@@ -4,29 +4,81 @@ about: Use this template for feature and enhancement proposals.
-3. After discussion & agreement on feature request, start implementation
+## Proposal Summary
 
+(In a few sentences, provide a clear, high-level description of the feature request)
 
-## Describe the proposal
-Provide a clear high-level description of the feature request in the following sections. Feature requests that are likely to be accepted:
-* Are minimal in scope (note that it's always easier to add additional functionality later than remove functionality)
-* Are extensible (e.g. if adding an integration with an ML framework, is it possible to add similar integrations with other frameworks?)
-* Have user impact & value that justifies the maintenance burden of supporting the feature moving forwards. The [JQuery contributor guide](https://contribute.jquery.org/open-source/#contributing-something-new) has an excellent discussion on this.
+## Motivation
+- What is the use case for this feature?
+- Why is this use case valuable to support for MLflow users in general?
+- Why is this use case valuable to support for your project(s) or organization?
+- Why is it currently difficult to achieve this use case? (please be as specific as possible about why related MLflow features and components are insufficient)
 
-### Motivation
-What is the use case in mind?  Why is it valuable to support, and why is it currently difficult or impossible to achieve? Could the desired functionality alternatively be implemented as a third-party package using MLflow public APIs?
+## Proposed Changes
 
-### Proposed Changes
-For user-facing changes, what APIs are you proposing to add or modify? For internal changes, what code paths will need to be modified?
+### Can this feature be introduced as an MLflow Plugin?
+[MLflow Plugins] enable integration of third-party modules with many of MLflow’s components, allowing you to maintain and iterate on certain features independently of the MLflow Repository.
+
+Please refer to the https://mlflow.org/docs/latest/plugins.html for a list of supported MLflow Plugin types. Afterwards, select the option that best applies and fill in the **Explanation** field if applicable:
+
+- [ ] Yes. I think this feature can be introduced as an MLflow Plugin.
+- [ ] No. It does not make sense to structure this feature as an MLflow Plugin. **Explanation:**
+- [ ] No. I don’t think MLflow Plugins currently supports this type of feature. **Explanation:**
+
+If you selected “Yes,” please submit this feature request and add the “Plugin” GitHub label. An MLflow Community member will provide further guidance on how to get started with developing this feature as an MLflow Plugin.
+
+If you selected “No”, please complete the subsequent sections.
+
+### Which MLflow component(s) does this feature affect?
+
+- [ ] UI
+- [ ] CLI
+- [ ] API
+- [ ] REST-API
+- [ ] Examples
+- [ ] Docs
+- [ ] Tracking
+- [ ] Projects
+- [ ] Artifacts
+- [ ] Models
+- [ ] Model Registry
+- [ ] Scoring
+- [ ] Serving
```

Comment:
@sueann Can we dedupe `Scoring` and `Serving` here and in the pull request template?

Reply:
Maybe also API and REST-API? Dunno how a user would choose between those.

Reply:
sounds good - i think we want to just keep "Scoring"

Reply:
Thanks for the input! I've consolidated `Serving` and `Scoring` to just `Scoring`. I've consolidated `API` and `REST-API` as just `API`.